administration/src/cards/PdfFactory.ts

```diff
@@ -29,18 +48,20 @@ const fillContentAreas = async (
   deepLink: string,
   region?: Region
 ): Promise<void> => {
-  const helveticaFont = await doc.embedFont(StandardFonts.Helvetica)
+  const font = await doc.embedFont(StandardFonts.Helvetica)
   pdfConfig.elements?.dynamicActivationQrCodes.forEach(configOptions =>
     pdfQrCodeElement(configOptions, { page: templatePage, qrCode: dynamicCode })
   )
 
-  const helveticaBoldFont = await doc.embedFont(StandardFonts.HelveticaBold)
+  const fontBold =
+    (pdfConfig.customBoldFont && (await loadCustomFont(pdfConfig.customBoldFont, doc))) ||
+    (await doc.embedFont(StandardFonts.HelveticaBold))
```

Comment:
```suggestion
  const fontBold =
    pdfConfig.customBoldFont && (await loadCustomFont(pdfConfig.customBoldFont, doc)) ||
    await doc.embedFont(StandardFonts.HelveticaBold)
```
For me it also works with less parenthesis

Reply:
yes it will work with less parenthesis, because in js && is "stronger" than ||. But most humans do not know this on the first sight, so the parentheses increase readability as it is clear at the first look what belongs together

Reply:
alright i'm fine to keep it